src/server/components/dashboard/listsItems/controllers.ts

```diff
@@ -754,11 +754,9 @@ export async function listItemEditRequestValidation(
       },
     });
   } else if (!userIsListPublisher(req, list)) {
-    res.status(403).send({
-      error: {
-        message: "User doesn't have publishing right on this list",
-      },
-    });
+    res.status(403);
+    logger.error("User doesn't have publishing right on this list");
```

Comment:
if you move this logging message to the line above you can set the status inline with the call to the render function

Reply:
Ah cool, that's good to know 👍